### PR TITLE
feat(frontend): Add AutoComplete component (#34)

### PR DESCRIPTION
**Pull request by Rolly007**

### CODE REVIEW

File: frontend/src/components/MoviesPagination/MoviesPagination.jsx [hunk 1]
```diff
@@ -5,21 +5,28 @@ import Typography from '@material-ui/core/Typography';
...
 import MovieCard from 'components/MovieCard';
 import Pagination from 'components/Pagination';
-import { getCategoryDisplayName } from 'utils';
+import { getCategoryDisplayName, defaultHeaders } from 'utils';
 
 import styles from './MoviesPagination.module.scss';
 
-function MoviesPagination({ category, useAxiosHook }) {
+function MoviesPagination({
+  category,
```
Comment (revodark) [inline frontend/src/components/MoviesPagination/MoviesPagination.jsx]: Maybe you could update the propTypes

File: frontend/src/components/MoviesPagination/MoviesPagination.jsx [hunk 2]
```diff
@@ -5,21 +5,28 @@ import Typography from '@material-ui/core/Typography';
...
   const config = {
-    url: '/movies',
+    url: requestPath,
+    headers: defaultHeaders,
     method: 'get',
     params: {
       category,
-      sortBy: 'popularity.desc',
+      list,
+      sortBy: category ? 'popularity.desc' : order,
```
Comment (Rodri77) [inline frontend/src/components/MoviesPagination/MoviesPagination.jsx]: wouldn't it be better if we have "popularity.desc" as default value for order prop, also I was thinking we can rename this to sortBy so we can directly provide the prop value and it's easier to follow code-wise , as a question do we need to validate if we have a category for this?

File: frontend/src/views/MovieCategory/MovieCategory.jsx [hunk 3]
```diff
@@ -6,11 +6,17 @@ import MoviesPagination from 'components/MoviesPagination';
 
 import styles from './MovieCategory.module.scss';
 
+const requestPath = '/movies';
+
 const MovieCategory = ({ category, useAxiosHook }) => {
```
Comment (Rodri77) [inline frontend/src/views/MovieCategory/MovieCategory.jsx]: We can have default return for this component

File: frontend/src/components/AutoComplete/AutoComplete.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import styles from './AutoComplete.module.scss';
+
+const AutoCompleteMenu = ({
+  autoCompleteParams,
+  menuLabel,
+  onChangeFunction,
+  disableClearOption = true
+}) => (
+  <Autocomplete
```
[the remark this answers is not in the document]
Comment (Rolly007) [reply frontend/src/components/AutoComplete/AutoComplete.jsx]: ok thanks

**Pull request merged**